jsonpaths/com.olark/transcript_1.json

```diff
@@ -0,0 +1,42 @@
+{
+    "jsonpaths": [
+        "$.schema.vendor",
+        "$.schema.name",
+        "$.schema.format",
+        "$.schema.version",
+        "$.hierarchy.rootId",
+        "$.hierarchy.rootTstamp",
+        "$.hierarchy.refRoot",
+        "$.hierarchy.refTree",
+        "$.hierarchy.refParent",
+        "$.data.groups",
+        "$.data.id",
+        "$.data.items",
+        "$.data.kind",
+        "$.data.operators..*.emailAddress",
```

Comment:
I'm very surprised this was generated by Igluctl and don't think this is going to work.

Reply:
What is igluctl supposed to create in this case (so that I can open an issue there)?

Reply:
I think it should fail with something like `Unsupported characters` error.